docs/user-guide/node-selection/pod-with-pod-affinity.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-newlabels
+  annotations:
+    scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
How do we validate that annotations are sane?  It seems we would want an analysis tool that could check if your annotation is valid or could ever be met.